### ISSUE TITLE
Accuracy don't change after many epoch

### ISSUE THREAD

**Issue by henryccl** [rcouturier/steganalysis_with_CNN_and_SRM#3]
Hello, I'm sorry to borther you.While I'm trianing my dataset(S-Uinward payload=0.4),the accuracy don't change.
After several epochs, the accuracy of each step is the same as the previous epoch,and in every epoch the cnn get the same result.
Seems like the cnn was not working,I use batch_size=64, seed=2,epochs = 2000,and I don't change anything else in you code conv_stego20.py.What's going wrong?What should I do?The following is a record of the accuracy.
[accuracy_record.txt](https://github.com/rcouturier/steganalysis_with_CNN_and_SRM/files/3301490/accuracy_record.txt)

**Comment by rcouturier** [comment]
Hello,

In fact, this is just because you don't wait enough. Here is a log of a script we ran during our experiments. We have used many scripts like this
[res20_cluster3_1key.240169.txt](https://github.com/rcouturier/steganalysis_with_CNN_and_SRM/files/3301686/res20_cluster3_1key.240169.txt)

**Comment by henryccl** [comment]
Hello，
actually I think, there must be something wrong ,cause firstly I 
have trained more than 80 epochs but the accuracy is same ,while your 
accuracy get slowly increase .In 80 epoch , your accuracy 
get to 0.7 but my is still 0.49（it still get 0.49 in epoch 200）. Secondly, the Confusion_matrix 
of you looks like more normal[[4484 2073], [ 511 2916]] ,while my 
Confusion_matrix is [[0 0],[ 4991 4993]] or [[4991 4993],[0 0]]. It seems like CNN is 
making the same prediction all the time. Where did I get it wrong? What should I do?

**Comment by rcouturier** [comment]
Are you sure that your images are correctly generated? Did you check your cover and stego images?

**Comment by henryccl** [comment]
My stego get something wrong .Now I have finished training the convolutional neural network, thank you for your help all the time

**Comment by rcouturier** [comment]
Good to see that it works now.

**Comment by doronruas** [comment]
Hey, Can you please post the network saved file? It takes so much time to create one by myself.

**Comment by rcouturier** [comment]
Hello,

In fact, as this work is quite old, I didn't save all the weight...
Sorry for that.

Raphaël
